index.html

```diff
@@ -0,0 +1,182 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bierproductie: Info Pages</title>
+    <link rel="stylesheet" href="styles/main.css"/>
+    <script src="scripts/script.js" defer></script>
+    <script src="scripts/maintain.js" defer></script>
+    <script src="https://code.iconify.design/1/1.0.7/iconify.min.js"></script>
+  </head>
+  <body>
+
+    <!-- Side navigation -->
+    <div class="sidenav">
+      <span class="iconify marked-page" onclick="location.href='/';" data-icon="cil:touch-app" data-inline="false">
+      </span>
+      <span class="iconify" onclick="location.href='/settings.html';" data-icon="carbon:settings" data-inline="false">
```

Comment:
onclick does not redirect to settings.html in Firefox browser. 
In Chrome it tries to find at this location: file:///D:/settings.html (my git folder is on my D drive, but instead of using the same folder path as index.html, it goes to the root).

Reply:
I have not run it without hosting it, I am gonna check if I can fix it